frontend/scss/pages/_course.scss

```diff
@@ -1,4 +1,5 @@
 .green-section {
+  margin-top: 60px;
```

Comment:
@andela-iakande for consistency would it be better if ?
#course-page {
    /* margin-top: 0; */
}

therefore:
 
.page-wrapper {
    margin-top: 65px;
}